confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+####################
+### DeviceGroup Configuration
+####################
+
+# Number of DeviceGroups per StorageGroup
+# Datatype: int
+# device_group_count=10000
+
+# DeviceGroup hash algorithm
+# Datatype: String
+# Currently, BKDR, AP, JS or SDBM hash algorithm is available
+# device_group_hash_algorithm=BKDR
+
+# DeviceGroupHashExecutor package name
+# Datatype: String
+# If you want to implement your own hash algorithm, you can inherit the DeviceGroupHashExecutor class and
+# modify this parameter to correspond to your Java package prefix.
+# But make sure that your own DeviceGroupHashExecutor class name is device_group_hash_algorithm + "HashExecutor".
+# device_group_hash_executor_package=org.apache.iotdb.confignode.manager.hash.
```

Comment:
hi,  we could merge a config parameter. example:
device_group_hash_algorithm_class=org.apache.iotdb.confignode.manager.hash.BKDRHashExecutor.
2. remove device_group_hash_executor_package parameter

Reply:
Good idea! I'm done.